app.py

```
@@ -20,6 +21,22 @@
+    try:
+        error_data = request.get_json()
+        if not error_data or 'error' not in error_data or 'log_error' in str(error_data):
+            return jsonify({'status': 'error', 'message': 'Invalid data'}), 400
+        
+        os.makedirs('logs', exist_ok=True)
+        
+        with open('logs/logging.txt', 'a') as f:
+            f.write(error_data['error'])
+            
+        return jsonify({'status': 'success'})
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)